src/conditional-flow/multiple-conditions.js

```diff
@@ -2,20 +2,26 @@
 // than or equal to lower AND less than or equal to upper.
 // Implement this with a single condition.
 function isInRange (num, lower, upper) {
-
-  // TODO: write code in this function body to pass the tests
+if (num >= lower && num <= upper) {
```

Comment:
This can be simplified. `num >= lower && num <= upper` creates a boolean value representing if num is greater than or equal to lower AND num is less than or equal to upper. So we can return that value like so:
```js
return num >= lower && num <= upper
```